services/invitations/tests/unit/test_core_settings.py

```diff
@@ -2,7 +2,13 @@
 # pylint: disable=unused-argument
 # pylint: disable=unused-variable
 # pylint: disable=too-many-arguments
+"""
```

Comment:
I find a bit too much to repeat this everywhere. can this not go in the pytest cli --help?